content/guides/compose-bake/index.md

```diff
@@ -0,0 +1,396 @@
+
+Did you know that you can also use Compose just to build the service images?
+The `docker compose build` command lets you invoke a build using the build
+configuration specified in the Compose file. For example, to build the `vote`
+service with this configuration, run:
+
+```console
+$ docker compose build vote
+```
+
+Omit the service name to build all services at once:
+
+```console
+$ docker compose build
+```
+
+The `docker compose build` command is useful when you only need to build images
+without running services.
+
+The Compose file format supports a number of properties for defining your
+build's configuration. For example, to specify the tag name for the images, set
+the `image` property on the service.
+
+```yaml
+services:
+  vote:
+    image: username/vote
+    build:
+      context: ./vote
+      target: dev
+    #...
+
+  result:
+    image: username/result
+    build:
+      context: ./result
+    #...
+
+  worker:
+    image: username/worker
+    build:
+      context: ./worker
+    #...
+```
+
+Running `docker compose build` creates three service images with fully
+qualified image names that you can push to Docker Hub.
+
+The `build` property supports a [wide range](/reference/compose-file/build.md)
+of options for configuring builds. However, building production-grade images
+are often different from images used in local development. To avoid cluttering
+your Compose file with build configurations that might not be desirable for
+local builds, consider separating the production builds from the local builds
+by using Bake to build images for release. This approach separates concerns:
+using Compose for local development and Bake for production-ready builds, while
+still reusing service definitions and fundamental build configurations.
+
+## Build with Bake
+
+Like Compose, Bake parses the build definition for a project from a
+configuration file. Bake supports HashiCorp Configuration Language (HCL), JSON,
+and the Docker Compose YAML format. When you use Bake with multiple files, it
+will find and merge all of the applicable configuration files into one unified
+build configuration. The build options defined in your Compose file are
+extended, or in some cases overridden, by options specified in the Bake file.
+
+The following section explores how you can use Bake to extend the build options
+defined in your Compose file for production.
+
+### View the build configuration
+
+Bake automatically creates a build configuration from the `build` properties of
+your services. Use the `--print` flag for Bake to view the build configuration
+for a given Compose file. This flag evaluates the build configuration and
+outputs the build definition in JSON format.
+
+```console
+$ docker buildx bake --print
+```
+
+The JSON-formatted output shows the group that would be executed, and all the
+targets of that group. A group is a collection of builds, and a target
+represents a single build.
+
+```json
```

Comment:
My attempt had slightly different output (`buildx` verision `github.com/docker/buildx v0.17.1-desktop.1 1a2ee7e8d945b88fdc18f1948da491c2f540dec9`)

```json
{
  "group": {
    "default": {
      "targets": [
        "vote",
        "result",
        "worker",
        "seed"
      ]
    }
  },
  "target": {
    "result": {
      "context": "result",
      "dockerfile": "Dockerfile",
      "network": ""
    },
    "seed": {
      "context": "seed-data",
      "dockerfile": "Dockerfile",
      "network": ""
    },
    "vote": {
      "context": "vote",
      "dockerfile": "Dockerfile",
      "target": "dev",
      "network": ""
    },
    "worker": {
      "context": "worker",
      "dockerfile": "Dockerfile",
      "network": ""
    }
  }
}

Reply:
Yeah... I might have doctored the example slightly, and then my formatter messed with it further. Probably best to keep it porcelain for this guide 👍🏻 Will change.

Reply:
`"network": ""` should not happen anymore since v0.17.1